pkg/service/lb.go

```diff
@@ -106,23 +106,57 @@ func lbTags(app string, process string) map[string]string {
 	}
 }
 
+type loadBalancerExposureError struct {
```

Comment:
I take that back, we're not in package lb. `LoadBalancerExposureError` would be fine imo.